Packs/Gmail/Integrations/Gmail/Gmail.yml

```diff
@@ -224,6 +246,13 @@ script:
       - metadata
       - minimal
       - raw
+    - auto: PREDEFINED
+      defaultValue: False
+      description: Wether ro run gmail-get-attachments as well or not.
+      name: run-get-attachments-command
```

Comment:
Let's not name the argument as a second command but something like `include-attachments` or `get-attachments`.
I suggest you to consult our TPMs about it.